src/generic/dirctrlg.cpp

```diff
@@ -389,11 +439,14 @@ bool wxGenericDirCtrl::Create(wxWindow *parent,
     long treeStyle = wxTR_HAS_BUTTONS;
 
     treeStyle |= wxTR_HIDE_ROOT;
+    m_availableID = 1;
 
 #ifdef __WXGTK20__
     treeStyle |= wxTR_NO_LINES;
 #endif
 
+    m_style = style;        // Keep a record of the style, which we'll need in HandleDirMenu()
```

Comment:
Style is stored in `m_windowStyle`, accessible by `GetWindowStyleFlag()` and convenient `HasFlag()`.

Reply:
Cool! Nice!